src/sections/hello-dataverse/HelloDataverse.tsx

```diff
@@ -0,0 +1,26 @@
+import logo from './logo.svg'
+import styles from './HelloDataverse.module.scss'
+import { Trans, useTranslation } from 'react-i18next'
+
+export function HelloDataverse() {
+  const { t } = useTranslation('helloDataverse')
+
+  return (
+    <div className={styles.container}>
+      <header>
+        <h1 className={styles.title}>{t('title')}</h1>
+        <img src={logo} className={styles.logo} alt={t('altImage') ?? 'logo'} />
+        <p>
+          <Trans t={t} i18nKey="description" components={{ 1: <code /> }} />
```

Comment:
This Trans syntax is a little rough but I'm glad to see that for a simple case like `{t('title')}` it looks nice. 😄 

Here I see you're substituting a `<code>` tag. Fancy. 🧐  And a little ugly. 😜

Reply:
Definitely, I didn't like the Trans syntax neither but among all the [alternatives](https://react.i18next.com/latest/trans-component) for the `<code>` tag I liked that one the most

Mainly because if I don't know the text behind `{t('title')}` then I don't want to know the text behind `i18nKey="description"` neither.

The alternative looks like this where the text can be seen in plain English:

```
<Trans t={t} i18nKey="description">
      Edit <code>src/sections/hello-dataverse/HelloDataverse.tsx</code> and save to reload.
</Trans>
```

Maybe it looks more beautiful to you this way, I'm open to change this